test/gtest/ucp/test_ucp_proto.cc

```diff
@@ -431,16 +441,118 @@ class test_proto_perf : public ucs::test {
         return ucp_proto_perf_find_segment_lb(perf.get(), start);
     }
 
-    void expect_empty_range(size_t start, size_t end)
+    static const ucp_proto_flat_perf_range_t *
+    find_lb(flat_perf_ptr_t flat_perf, size_t start)
```

Comment:
const

Reply:
Do you want me to remove static to make them const? It would cause removing static from functions like `get_sample_points` too.